cxxprocessor/Dockerfile

```diff
@@ -42,4 +42,4 @@ EXPOSE 4004/tcp
 
 WORKDIR /project/simplewallet/cxxprocessor
 
-CMD bash -c "./build.sh && ./bin/simplewallet-tp"
+CMD bash -c "./build.sh && ./build/bin/simplewallet-tp"
```

Comment:
I tested this and the last line needs to be:
`
CMD bash -c "./build.sh && ./build/bin/simplewallet-tp tcp://validator:4004"
`
I think this is because the `docker-compose.yaml` file uses an older version of `simplewallet-tp`, so it needs a command line option to work for now (until it's updated):

`   image: askmish/sawtooth-simplewallet-tp-cxx:latest`